src/app/nonprofit/nonprofit-page/donation-dialog.html

```diff
@@ -0,0 +1,124 @@
+<button mat-icon-button class="close-icon-button" [mat-dialog-close]="true">
+    <mat-icon class="close-icon">close</mat-icon>
+</button>
+<h1 mat-dialog-title>
+    Thank you for choosing to to donate to {{data.nonprofitName}}!
+</h1>
+<div mat-dialog-content class="donation-stepper-container">
+    <mat-vertical-stepper linear #stepper>
```

Comment:
I made the stepper vertical because it flows better when the screen changes sizes. So for example, when the user makes the screen smaller, the horizontal stepper looks really weird but the vertical one handles it better. This is just planning ahead in case we make it mobile-friendly since the content inside the modal will be easier to handle.

Reply:
Ahh gotcha, that makes sense then. Are you able to align the positioning of the next button in relation to the input so that it looks a bit more streamlined? I think the positioning of the next button could be fine but the centered donation input with the left aligned next button in different rows looks off to me, but that could also just be a personal thing

Reply:
So I actually had it like that at first but I changed all the buttons to be aligned to the left side so that all of the steppers would have the buttons on the left, and all of the inputs are aligned at the center in columns (i.e. second stepper has two columns aligned at the center)